### PR TITLE
Add the ability to customize section and field names on save/load.

### LINKED ISSUE #1: Possibility to change section name
How I can change section name to any human-readable form?

Example (name in result ini-file is "const(config.Config)"):

``` d
import inifiled;


@INI("Main section")
private struct Config {
    @INI("The lastname of the Person") string lastname;

    @INI("The height of the Person") float height;

    @INI("Some strings with a very long long INI description that is longer" ~
        " than eigthy lines hopefully."
    ) string[] someStrings;
}

const Config cfg;

static this()
{
    cfg.lastname = "test name";

    writeINIFile(cfg, "config.conf");
}
```

``` ini
$ cat config.conf 
; Main section
[const(config.Config)]
; The lastname of the Person
lastname="test name"
; The height of the Person
height="nan"
; Some strings with a very long long INI description that is longer than eigthy
;  lines hopefully.
someStrings=""
```

### PR DESCRIPTION
**Pull request by Hackerpilot**
Fixes #1. Also fixes some spelling mistakes.

### CODE REVIEW

**Comment by ghost** [comment]
@burner, if you take this PR please also push a new git tag for DUB. We actually require it for D-Scanner.

File: source/inifiled.d [hunk 1]
```diff
@@ -366,31 +364,43 @@ void writeINIFile(T)(ref T t, string filename) @trusted {
...
 		   isSomeString;
 	import std.format : formattedWrite;
 	if(hasUDA!(T, INI) && section) {
-		writeComment(oRange, getINI!T());
+		writeComment(oRange, getINI!T().msg);
 	}
 
 	if(section) {
-		oRange.formattedWrite("[%s]\n", getTypeName!T);
+		if (hasUDA!(T, INI))
```
Comment (burner) [inline source/inifiled.d]: could you move the { to the end of the line

File: source/inifiled.d [hunk 2]
```diff
@@ -366,31 +364,43 @@ void writeINIFile(T)(ref T t, string filename) @trusted {
...
+		{
+			auto ini = getINI!(T);
+			if (ini.name !is null)
+				oRange.formattedWrite("[%s]\n", ini.name);
+			else
```
Comment (burner) [inline source/inifiled.d]: could you please add curly braces for the two else statements

**Comment by burner** [comment]
apart of the nitpicks it looks good. thank you.
As soon as these are fixed I'm gone merge and push a new tag.

**Review by burner — CHANGES_REQUESTED** [review]

**Review by burner — APPROVED** [review]

**Comment by burner** [comment]
@Hackerpilot thank you

**Comment by burner** [comment]
new tag is pushed

**Pull request merged**